packages/ui/src/proposals/components/ProposalHistory.tsx

```diff
@@ -21,9 +21,8 @@ export const ProposalHistory = ({ proposal }: ProposalHistoryProps) => {
     const steps = [createdStatus, ...updates, ...(endStatus || [])]
 
     return steps.map(({ status, inBlock }, index) => ({
-      title: status,
+      title: `${status} in block ${inBlock.number}`,
       type: index === steps.length - 1 ? 'active' : 'past',
```

Comment:
About this part I replied in #3966.

But please reset this file (`git checkout dev -- packages/ui/src/proposals/components/ProposalHistory.tsx`) and implement the changes in another PR

Reply:
sorry for the delay: bc91735